setup.cfg

```diff
@@ -80,7 +80,7 @@ install_requires =
     Jinja2 >= 2.10.1
     pexpect >= 4.6.0, < 5
     psutil == 5.4.6; sys_platform!="win32" and sys_platform!="cygwin"
-    PyYAML == 3.13
+    PyYAML >= 5.1
```

Comment:
Can we also get a `< 6` ? Assuming semantic versioning ... but we should make sure :+1:

Reply:
Please see https://github.com/yaml/pyyaml/blob/master/CHANGES#L7.

We need to put an upper bound on this, no?